src/transformers/dynamic_module_utils.py

```diff
@@ -395,8 +395,8 @@ def custom_object_save(obj, folder, config=None):
             "this code in a separate module so we can include it in the saved folder and make it easier to share via "
             "the Hub."
         )
-    # Add object class to the config auto_map
-    if config is not None:
+
+    def _set_auto_map_in_config(_config):
```

Comment:
We need to apply that code to one or several configs, so putting it in an internal function.